staging/src/k8s.io/apiserver/plugin/pkg/audit/truncate/truncate.go

```diff
@@ -129,7 +129,7 @@ func truncate(e *auditinternal.Event) *auditinternal.Event {
 
 func (b *backend) Run(stopCh <-chan struct{}) error {
 	// Nothing to do here
-	return nil
```

Comment:
is this a general pattern that a backend starts it's delegates? Do we do that elsewhere as well in auditing? Feels a bit odd.

Reply:
For example, that's how we are doing in union (it's not an ordinary backend): https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/audit/union.go#L46

Also the same we have in buffered backend:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go#L121

About other places in the apiserver I'm not sure.

If we imagine that structure of backends is a tree, then it makes complete sense to start sub-backends from each node of this tree.

Reply:
/lgtm

Reply:
If it's uniform in the audit backend, it's fine.